src/api/models/nls/ncn_reboot.go

```diff
@@ -2,7 +2,7 @@
  *
  *  MIT License
  *
- *  (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+ *  (C) Copyright 2022,2024 Hewlett Packard Enterprise Development LP
```

Comment:
I think there should be a hyphen instead of comma. (2022-2024) .
Same goes for other files as well